### PR TITLE
[DOCS] EditorInterface

### PR DESCRIPTION
**Pull request by zerastos**

### CODE REVIEW

File: doc/classes/EditorInterface.xml [hunk 1]
```diff
@@ -89,6 +102,7 @@
 			<argument index="1" name="preview_size" type="int">
 			</argument>
 			<description>
+				Returns mesh previews rendered at the given size as an [Texture] [Array].
```
Comment (Noshyaar) [inline doc/classes/EditorInterface.xml]: - an [Array] of [Texture]\\ s or
- a [Texture] [Array]?

File: doc/classes/EditorInterface.xml [hunk 2]
```diff
@@ -121,6 +138,7 @@
 			<argument index="1" name="with_preview" type="bool" default="true">
 			</argument>
 			<description>
+				Saves the scene as a file at [code]path[/code]\ .
```
Comment (Noshyaar) [inline doc/classes/EditorInterface.xml]: `[/code].`

File: doc/classes/EditorInterface.xml [hunk 3]
```diff
@@ -15,60 +17,70 @@
 			<argument index="0" name="resource" type="Resource">
 			</argument>
 			<description>
+				Edits the given [Resource]\ .
```
Comment (Noshyaar) [inline doc/classes/EditorInterface.xml]: is `[Class].` not working?

**Comment by Fallayn** [comment]
Fixed!

File: doc/classes/EditorInterface.xml [hunk 4]
```diff
@@ -15,60 +17,70 @@
...
 			</return>
 			<description>
+				Returns the [EditorSettings].
 			</description>
 		</method>
 		<method name="get_editor_viewport">
 			<return type="Control">
 			</return>
 			<description>
+				Returns the editor viewport.
```
Comment (cbscribe) [inline doc/classes/EditorInterface.xml]: I'm not sure about this one. The return type says `Control` - what object is actually being returned?

File: doc/classes/EditorInterface.xml [hunk 5]
```diff
@@ -79,6 +91,7 @@
 			<argument index="1" name="for_property" type="String" default="&quot;&quot;">
 			</argument>
 			<description>
+				Inspects the given [code]object[/code] for the given property.
```
Comment (cbscribe) [inline doc/classes/EditorInterface.xml]: I don't understand the purpose of this method. It doesn't return anything - what does it actually do?

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/EditorInterface.xml (hunk 4 above)
Comment (Fallayn) [reply doc/classes/EditorInterface.xml]: It returns `Viewport` or `NULL`, just tested this.

File: doc/classes/EditorInterface.xml (hunk 5 above)
Comment (Fallayn) [reply doc/classes/EditorInterface.xml]: It selects the given object in the Editor's inspector and scrolls to the given property, if it exists. The programmatic equivalent of clicking on a node in the scene hierarchy.

**Comment by Fallayn** [comment]
@cbscribe Hope those two places are more clear now :)

**Comment by Fallayn** [comment]
Somehow, GitHub didn't update the PR. This is the current version of this PR, squashed as 1 commit:
https://github.com/Fallayn/godot/commit/7f2d36bf843fca339c0cadc5800ea0013b6eeff8

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Looks good, now you just need to update the PR.

**Comment by Fallayn** [comment]
@cbscribe Was a GitHub issue, the PR got updated now. (https://twitter.com/githubstatus/status/920310391820947458)

**Pull request merged**